src/components/Header/Header.tsx

```diff
@@ -0,0 +1,143 @@
+import styles from './header.module.css';
+
+function Header() {
+    return (
+        <header className={styles.header}>
+            <div className={styles.headerContainer}>
+                <a href="./">
+                    <svg className={styles.headerLogoIcon} xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0 0 40 40">
+                        <path
+                            fillRule="evenodd"
+                            clipRule="evenodd"
+                            d="M38.6957 20C38.6957 30.3253 30.3253 38.6957 20 38.6957C9.67468 38.6957 1.30435 30.3253 1.30435 20C1.30435 9.67468 9.67468 1.30435 20 1.30435C30.3253 1.30435 38.6957 9.67468 38.6957 20ZM40 20C40 31.0457 31.0457 40 20 40C8.9543 40 0 31.0457 0 20C0 8.9543 8.9543 0 20 0C31.0457 0 40 8.9543 40 20ZM20 8.69564L24.5423 17.4825L29.5652 13.4066V27.8261H10.4347V13.4066L15.4576 17.4825L20 8.69564ZM28.3859 24.8971V15.8235L25.0665 18.5066L28.3859 24.8971ZM23.6251 18.2199L20 11.215L16.3748 18.2199L20 21.1693L23.6251 18.2199ZM11.5922 24.8971L14.9116 18.5066L11.5922 15.8235V24.8971ZM15.8288 19.2645L11.9416 26.7405H28.0583L24.1711 19.2645L20 22.6441L15.8288 19.2645Z"
+                        />
+                    </svg>
+                </a>
+
+                <div className={styles.headerThemeModeBtns}>
+                    <button className={styles.headerThemeModeBtn}>
+                        <svg
+                            className={styles.headerThemeModeBtnIcon}
+                            xmlns="http://www.w3.org/2000/svg"
+                            width="20"
+                            height="20"
+                            viewBox="0 0 20 20"
+                        >
+                            <g clipPath="url(#clip0_3001_1631)">
+                                <path
+                                    d="M10.0001 3.33329V1.66663M10.0001 16.6666V18.3333M5.34526 5.34514L4.16675 4.16663M14.7734 14.7733L15.952 15.9518M3.33341 9.99996H1.66675M16.6667 9.99996H18.3334M14.7738 5.34514L15.9524 4.16663M5.34567 14.7733L4.16716 15.9518M10.0001 14.1666C7.69889 14.1666 5.83341 12.3011 5.83341 9.99996C5.83341 7.69877 7.69889 5.83329 10.0001 5.83329C12.3013 5.83329 14.1667 7.69877 14.1667 9.99996C14.1667 12.3011 12.3013 14.1666 10.0001 14.1666Z"
+                                    strokeWidth="1.5"
+                                    strokeLinecap="round"
+                                    strokeLinejoin="round"
+                                />
+                            </g>
+                        </svg>
+                    </button>
+
+                    <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 2 20">
+                        <path d="M1 0V20" stroke="#656565" />
+                    </svg>
+
+                    <button className={[styles.headerThemeModeBtn, styles.headerThemeModeBtnActive].join(' ')}>
```

Comment:
Creating a utility function to generate a string from an array of classes would be helpful.

Reply:
Fixed